pkg/monitoring/mimir/querier/querier.go

```diff
@@ -0,0 +1,53 @@
+package querier
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/prometheus/client_golang/api"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/prometheus/common/model"
+)
+
+// QueryTSDBHeadSeries performs an instant query against Mimir.
+func QueryTSDBHeadSeries(ctx context.Context, clusterName string) (float64, error) {
+	config := api.Config{
+		Address: "http://mimir-gateway.mimir.svc/prometheus",
+	}
+
+	// Create new client.
+	c, err := api.NewClient(config)
+	if err != nil {
+		return 0, err
+	}
+
+	// Run query against client.
+	api := v1.NewAPI(c)
+
+	queryContext, cancel := context.WithTimeout(ctx, 2*time.Minute)
+	query := fmt.Sprintf("sum(max_over_time(prometheus_agent_active_series{cluster_id=\"%s\"}[6h]))", clusterName)
```

Comment:
Uses the active series in agents instead of prometheus tsdb head series so we can get rid of the MC proms